src/components/Menu.tsx

```diff
@@ -85,12 +85,14 @@ export default function UserMenu(props: UserMenuProps) {
               >
                 <div className="p-4 ">
                   {user?.isPro() ? <p>Cleanup Pro</p> : <p>Cleanup Free</p>}
-                  {user?.user && (
+                  {user && !user.user?.anonymous && (
```

Comment:
```suggestion
                  {user.user && !user.user.anonymous && (
```

If I correctly understand, the first `user` object is the `userContext` so can't be undefined. But `userContext.user` can be undefined during loading

Reply:
good catch, thanks!